docs/dev/cut-a-new-release.md

```diff
@@ -1,64 +1,230 @@
-   git push -u origin fixes-for-v1.19.1
-   ```
-4. Follow the [above steps](#glean-release-process) for cutting a new release, except that:
-    * When opening a PR to land the commits, target the `release-vX.Y` branch rather than master.
-    * When cutting the new release via GitHub's UI, target the `release-vX.Y` branch rather than master.
+1. If it does not exist yet, create a support branch from the version tag and push it:
+    ```
+    git checkout -b support/v24.0 v24.0.0
+    git push upstream support/v24.0
+    ```
+2. Create a hotfix branch for this support branch:
+    ```
+    git checkout -b hotfix-v24.0.1 support/v24.0
+    ```
+3. Fix the bug and commit the fix in one or more separate commits into your hotfix branch.
+4. Push your bug fixes and create a pull request against the support branch: <https://github.com/mozilla/glean/compare/support/v24.0...your-name:hotfix-v24.0.1?expand=1>
+5. When that pull request lands, wait for CI to finish on that branch and ensure it's green: <https://circleci.com/gh/mozilla/glean/tree/support/v24.0>
 
-> **Note:** Point-releases for older versions should be rarely required.
-> We support the last released version and fixes should go there whenever possible.
+### Finish a support branch
+
+1. Check out the support branch:
+    ```
+    git checkout support/v24.0
+    ```
+2. Update the changelog .
+    1. Add any missing important changes under the `Unreleased changes` headline.
+    2. Commit any changes to the changelog file due to the previous step.
+3. Run `bin/prepare-release.sh <new version>` to bump the version number.
+    1. The new version should be the next patch version of the support branch.
+    2. Let it create a commit for you.
+3. Push the support branch:
+    ```
+    git push upstream support/v24.0
+    ```
+4. Tag the release on GitHub:
+    1. [Draft a New Release](https://github.com/mozilla/glean/releases/new) in the GitHub UI (`Releases > Draft a New Release`).
+    2. Enter `v<myversion>` as the tag. It's important this is the same as the tags you put in the links in the changelog.
+    3. Select the support branch (e.g. `support/v24.0`) as the target.
+    3. Under the description, paste the contents of the release notes from `CHANGELOG.md`.
+5. Wait for the CI build to complete for the tag.
+    * You can check [on CircleCI for the running build](https://circleci.com/gh/mozilla/glean).
+6. Release the Rust crates:
+    ```
+    cd glean-core
+    cargo publish --verbose
+    cd ffi
+    cargo publish --verbose
+    ```
+7. Send a pull request to merge back any bug fixes to the development branch: <https://github.com/mozilla/glean/compare/master...support/v24.0?expand=1>
+    * This is important so that no changes are lost.
+    * This might have merge conflicts with the `master` branch, which you need to fix before it is merged.
+8. **DO NOT** delete the support branch.
+    * This branch stays around forever.
```

Comment:
Why is that?  Particularly since we (in most cases) only support the latest release?

Reply:
The support commits are not merged back into the release branch. If we would need _another hot fix_ on that version (and please don't let us need that) we could re-use the branch. It also serves as historic evidence.

There's no real technical reason to keep it around though. It does have a tag on it, so it can be found with all its bugfix commits.

I think I'm just as fine with deleting it

(for reference: the "hotfix old version" is not part of the original GitFlow, so I scraped this together from other recommendations)

Reply:
With that explanation, I guess I don't feel too strongly about this either way.  I leave it to you...